package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "vanilla-monorepo",
```

Comment:
Not necessarily in scope of this PR, but could be worth renaming this (and repo) to not include "vanilla" in the name.

Reply:
We can rename it to "DS-25" like we've been calling it recently, but I think it'll be less confusing to do that as a separate PR. I'll file one.

Reply:
Filed as #23